client/src/bastionlab/polars/remote_polars.py

```diff
@@ -289,64 +291,110 @@ def join_asof(
             ),
         )
 
-    def histplot(self: LDF, col_x: str, col_y: str, bins: int = 10, **kwargs):
-        model = ApplyBins(bins)
-        df = (
-            self.filter(pl.col(col_x) != None)
-            .filter(pl.col(col_y) != None)
-            .select([pl.col(col_y), pl.col(col_x)])
-            .apply_udf([col_x], model)
-            .groupby(pl.col(col_x))
-            .agg(pl.col(col_y).count())
-            .sort(col_x)
-            .collect()
-            .fetch()
-            .to_pandas()
-        )
-        sns.barplot(x=df[col_x], y=df[col_y], **kwargs)
+    def histplot(self: LDF, **kwargs):
```

Comment:
I would use explicit parameters for everything you use in the body and don't just forward to seaborn. That way, they will appear in the signature and be typed and the default value will also be explicit.